kalite/distributed/management/commands/screenshots.py

```diff
@@ -0,0 +1,338 @@
+import errno
+import json
+import os
+import glob
+import re
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandError
+from django.core.urlresolvers import reverse
+from django.utils.six import StringIO
+
+from selenium import webdriver
```

Comment:
See https://github.com/learningequality/ka-lite/wiki/Coding-guidelines-and-conventions#imports on how to organize import statements.